Typescript/TS/tscript.ts

```diff
@@ -0,0 +1,273 @@
+//Queryselectors
+const display = document.querySelector(".display-screen")! as HTMLInputElement;
+const btnMC = document.querySelector("#mc")! as HTMLButtonElement;
+const btnMR = document.querySelector("#mr")! as HTMLButtonElement;
+const btn2nd = document.querySelector("#btn2nd")! as HTMLButtonElement;
+const btnDeg = document.querySelector(".deg")! as HTMLButtonElement;
+const btnRad = document.querySelector(".rad")! as HTMLButtonElement;
+const buttons = document.querySelectorAll("button")!;
+const afr = document.querySelectorAll(".afr")!;
+const bfr = document.querySelectorAll(".bfr")!;
+getComputedStyle(document.documentElement).getPropertyValue('--color-blue');
+getComputedStyle(document.documentElement).getPropertyValue('--color-grey-deactive');
+getComputedStyle(document.documentElement).getPropertyValue('--color-white-button');
+getComputedStyle(document.documentElement).getPropertyValue('--color-black');
+
+//Functions
+function func_fact(){ //Factorial
+    let fact=1,i:number;
```

Comment:
Add spacing after `:` to all the variables where type is explicitly specified.
`let i: number;`